packages/react-native-editor/ios/gutenberg/GutenbergViewController.swift

```diff
@@ -315,6 +318,7 @@ extension GutenbergViewController {
         let cancelAction = UIAlertAction(title: "Keep Editing", style: .cancel)
         alert.addAction(toggleHTMLModeAction)
         alert.addAction(updateHtmlAction)
+        alert.addAction(unsupportedBlockUIAction)
```

Comment:
We don't have the equivalent for Android right? Not requesting it, just wondering if that can be a problem in your opinion

Reply:
That's right, we don't have this on Android. I think we don't have any of these actions, including `Toggle HTML mode`.
As far as I remember we can't since there's no bridge communication in the Android Demo app.

@hypest can confirm if this is still the case.

Reply:
What's not there in the Android demo apps is the "app bar" at the top that hosts those actions (on WPAndroid). The bridge and bridge communication present, just not exposed on an app bar.

Instead, we use the RN debug menu to expose those actions if needed... the "Show html" action is there for example. Implementation here: https://github.com/WordPress/gutenberg/blob/master/packages/react-native-editor/android/app/src/main/java/com/gutenberg/MainApplication.java#L247-L256